### PR TITLE
Add minimum NodeJS version requirement to package.json

### PR DESCRIPTION
**Pull request by mondeja**
**Issue:** #3959

Minimum version of NodeJS required to install simple-icons will be 0.12.18 after this pull.

### CODE REVIEW

File: package.json
Comment (ericcornelissen) [inline package.json:47]: I think this should be:

```suggestion
    "node": ">=0.12.18"
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
This is okay with me, if it turns out to be incorrect I guess we will find out soon enough :upside_down_face: 

Nevertheless, here are two language features we use that might potentially be missing from 0.12.18, though i wasn't able to verify this:

- The use of getters in icon objects

  https://github.com/simple-icons/simple-icons/blob/9cee719fcb487d7a43c9e4def631c82f8271c8bd/scripts/templates/icon-object.js#L5-L7

- The use of `enumerable: false` in index.js:

  https://github.com/simple-icons/simple-icons/blob/9cee719fcb487d7a43c9e4def631c82f8271c8bd/scripts/templates/index.js#L4

If you can verify this @mondeja that would be great. Otherwise, you can merge this after fixing the requested change and we will find out later whether or not these are (or something else is) a problem.

File: package.json
Comment (mondeja) [reply package.json]: What a mistake :sweat_smile: Sorry, it's fixed now. I will investigate these features before merge.

Comment (mondeja) [reply package.json]: - Getters are supported in 0.12.18 (tested manually and defined in [node.green](https://node.green/#ES2017-annex-b-Object-prototype-getter-setter-methods-__lookupGetter__)).
- `enumerable` property was been added at the beggining of the [`Object.defineProperty` support](https://node.green/#ES2017-annex-b-Object-prototype-getter-setter-methods-__lookupGetter__). More info [here](https://www.ecma-international.org/ecma-262/6.0/#sec-property-attributes).

**Pull request merged**
